remoulade/brokers/rabbitmq.py

```diff
@@ -45,14 +45,14 @@ class RabbitmqBroker(Broker):
 
     Examples:
 
-      >>> RabbitmqBroker(url="amqp://guest:guest@127.0.0.1:5672")
+      >>> RabbitmqBroker(url="amqp://guest:guest@127.0.0.1:5784")
```

Comment:
don't change anything here. Please create a separate PR for this topic, it will be easier